### DISCUSSION TITLE
Next JS Source Code Running Error in Dev

### DISCUSSION THREAD

**Discussion by balaganeshdev** [vercel/next.js#49287] [Help]
### Summary

Next JS Source Code Running Error in Dev

Please find the below commands:- 

1.  npx create-next-app@latest nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"

2.  npm run dev

![Next_JS_Error](https://user-images.githubusercontent.com/132655195/236437183-af00e989-cf19-43c3-b50a-eb2c664286fb.png)

**Comment by icyJoseph** [comment]
What Node.js version have you got installed? The `??=` syntax requires Node 15 IIRC.

**Comment by secan** [comment]
Following the answer by @icyJoseph (thank you), may I suggest to update the information provided at https://nextjs.org/learn/basics/create-nextjs-app/setup, where it says:

> If you don’t have Node.js installed, [install it from here](https://nodejs.org/en/). You’ll need Node.js version **10.13** or later.

**Reply by icyJoseph** [reply]
Yeah, I opened an issue on the repo most closely related: https://github.com/vercel/next-learn/issues/117#issue-1665862199

Bump it there :)

I'd say though, maybe the Next.js core team bandwidth must be quite full with everything going on. They're probably aware of various such issues but just can't catch a break to fix them.
